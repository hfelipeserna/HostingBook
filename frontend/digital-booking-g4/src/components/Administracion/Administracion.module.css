.camposObligatorios{
    color: var(--color-2);
    font-size: 0.875rem;
}
.containerPrincipal{
    background-color: var(--color-inputs-form-datos);
}

.formAdmin{
    background-color: var(--color-5);
    display: flex;
    flex-direction: column;
    align-items: center;
    width: 100%;
    border-radius: 10px;
    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
    border: 1px solid #dfe4ea;
    color: var(--color-2);
    margin: 2vh 0 64px 0;
    padding: 0 20px;
}

.subContainer{
    width: 100%;
    padding: 2vh 0;
}

.lineContainerInformacion{
    width: 100%;
    display: flex;
    justify-content: space-between;
    gap: 15px;
}

.containerTexto p{
    font-size: 0.875rem;
}

.containerSelect{
    display: flex;
    flex-direction: column;
    width: 100%;
    
}

.containerSelect label{
    color: var(--color-2);
    font-weight: 700;
    margin-top: 20px;
    cursor: pointer;
}

 .containerSelect select{
    /* reset */
    margin: 0;      
    -webkit-box-sizing: border-box;
    -moz-box-sizing: border-box;
    box-sizing: border-box;
    -webkit-appearance: none;
    -moz-appearance: none;

} 

option {
    font-size: 1rem ;
    color: var(--color-3);
    font-weight: 700;

}


select.minimal {
    /* styling */
    background-color: var(--color-5);
    border: 3px solid var(--color-5);
    box-shadow: 0px 1px 5px rgb(0 0 0 / 15%);
    color: var(--color-2);
    font-size: 1rem;
    padding: 1vh 40px 1vh 15px;
    width: 100%;
    margin-top: 5px;

    /* Cambiar la flecha del select solo con css*/
    background-image:
      linear-gradient(45deg, transparent 50%, var(--color-principal) 50%),
      linear-gradient(135deg, var(--color-principal) 50%, transparent 50%),
      linear-gradient(to right, var(--color-principal), var(--color-principal));
    background-position:
      calc(100% - 20px) calc(1em + 2px),
      calc(100% - 15px) calc(1em + 2px),
      calc(100% - 2.5em) 0.5em;
    background-size:
      5px 5px,
      5px 5px,
      1px 1.5em;
    background-repeat: no-repeat;
}
  
select.minimal:focus {
    background-image:
      linear-gradient(45deg, var(--color-principal) 50%, transparent 50%),
      linear-gradient(135deg, transparent 50%, var(--color-principal) 50%),
      linear-gradient(to right, var(--color-principal), var(--color-principal));
    background-position:
      calc(100% - 15px) 1em,
      calc(100% - 20px) 1em,
      calc(100% - 2.5em) 0.5em;
    background-size:
      5px 5px,
      5px 5px,
      1px 1.5em;
    background-repeat: no-repeat;
    border-color: var(--color-principal);
    outline: 0;
}
  
  
select:-moz-focusring {
    color: transparent;
    text-shadow: 0 0 0 #000;
}
 
/* Checkboxes de atributos*/
.checkboxes{
    background-color: var(--color-inputs-form-datos);
    border-radius: 5px;
    display: flex;
    flex-wrap: wrap;
    align-content: center;
    width: 100%;
    margin-top: 20px;
}

.checkboxes li{   
    width: 25%;
}


.checkboxes input[type="checkbox"]{
    display: none;
}

.checkboxes label {
    font-weight: 700;
    font-size: 1rem;
    padding: 5px 15px 5px 15px;
    gap: 15px;
    border-radius: 3px;
    cursor: pointer;
    transition: all 0.3s ease;
    margin: 5px 7px ;
    display: flex;
       
}

.checkboxes label i{
    color: var(--color-5);
    font-size: 1.25rem;
}
.checkboxes label span{
    color: var(--color-2);
}

.checkboxes label:hover {
    background-color: var(--color-inputs-form-datos);
    box-shadow: 0px 1px 5px rgb(0 0 0 / 15%);
    color: var(--color-2);
    
} 

.checkboxes label i:hover{
    color: var(--color-2);
}

.checkboxes input[type="checkbox"]:checked + label{
    padding: 5px 15px;
    background-color: var(--color-5);
    box-shadow: 0px 1px 5px rgb(0 0 0 / 15%);
    border-radius: 3px;
}

.checkboxes input[type="checkbox"]:checked + label i{
    color: var(--color-principal);
}  

@media (max-width: 768px) {
    .checkboxes li{   
        width: 50%;
    }
}
@media (max-width: 480px) {
    .containerSelect select{
        /* styling */
        width: 100%;
    }

    .lineContainerInformacion{
        display: flex;
        flex-direction: column;
        gap: 0;
    }

    .containerSelect label{
        margin-top: 7px;
    }

    .checkboxes li{   
        width: 100%;
    }
}
