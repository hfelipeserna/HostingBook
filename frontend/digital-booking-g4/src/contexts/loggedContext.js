import { createContext } from "react";

const loggedContext = createContext();

export default loggedContext;