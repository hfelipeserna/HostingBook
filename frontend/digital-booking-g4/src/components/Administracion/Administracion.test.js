import Administracion from "./Administracion"
import React from "react";
import "@testing-library/jest-dom/extend-expect";
import { render, screen } from "@testing-library/react"
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
import { createSerializer } from "enzyme-to-json";
import { configure, mount } from "enzyme";
import { MemoryRouter } from 'react-router-dom';
import loggedContext from '../../contexts/loggedContext'

configure({ adapter: new Adapter() });
expect.addSnapshotSerializer(createSerializer({ mode: "deep" }));
beforeEach(() => {
    jest.mock('react-router-dom', () => ({
        ...jest.requireActual('react-router-dom'),
        useHistory: () => ({
            push: jest.fn(),
            goBack: jest.fn()
        }),
    }));
})
describe("Administracion Input test", function () {
    test("Renderiza Por default", () => {
        const renderizacionSimple = render(
            <loggedContext.Provider value={{ Ã­sLogged: true }}>
                <MemoryRouter>
                    <Administracion></Administracion>
                </MemoryRouter></loggedContext.Provider>)
        expect(renderizacionSimple).toMatchSnapshot()

    });
});