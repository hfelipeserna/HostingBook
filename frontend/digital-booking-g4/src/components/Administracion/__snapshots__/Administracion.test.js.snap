// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Administracion Input test Renderiza Por default 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="contenedorHeader"
      >
        <div
          class="headerIzquierda"
        >
          <div
            class="a"
          >
            <span>
              <i
                aria-hidden="true"
                class="fas fa-chevron-left"
              />
            </span>
            <span>
              ATRÁS
            </span>
          </div>
        </div>
        <div
          class="headerDerecha"
        >
          <h3
            class="subtituloBloque favTitle"
          >
            Administración
          </h3>
        </div>
      </div>
      <section
        class="containerPrincipal"
      >
        <h3
          class="subtituloBloque undefined"
        >
          Crear propiedad
        </h3>
        <p
          class="camposObligatorios"
        >
          (Los campos identificados con * son obligatorios)
        </p>
        <form
          class="formAdmin"
        >
          <div
            class="subContainer"
          >
            <div
              class="lineContainerInformacion"
            >
              <div
                class="containerEstandarInput"
              >
                <label
                  class="labelEstandarInput"
                >
                  * Nombre de la propiedad
                </label>
                <div
                  class="sombraConAutocompletado"
                >
                  <input
                    class="estandarInput"
                    name="nombre"
                    required=""
                    type="text"
                    value=""
                  />
                </div>
              </div>
              <div
                class="containerSelect"
              >
                <label>
                  * Categoría
                </label>
                <select
                  class="minimal"
                  name="tituloCategoria"
                  required=""
                >
                  <option
                    class="optionDefault"
                    hidden=""
                    value="tituloCategoria"
                  />
                </select>
              </div>
            </div>
            <div
              class="lineContainerInformacion"
            >
              <div
                class="containerEstandarInput"
              >
                <label
                  class="labelEstandarInput"
                >
                  * Dirección
                </label>
                <div
                  class="sombraConAutocompletado"
                >
                  <input
                    class="estandarInput"
                    name="direccion"
                    required=""
                    type="text"
                    value=""
                  />
                </div>
              </div>
              <div
                class="containerSelect"
              >
                <label>
                  * CheckIn (horario min)
                </label>
                <select
                  class="minimal"
                  name="horarioCheckIn"
                  required=""
                >
                  <option
                    class="optionDefault"
                    hidden=""
                    value="horarioCheckIn"
                  />
                  <option
                    value="0"
                  >
                    00:00 AM
                  </option>
                  <option
                    value="1"
                  >
                    01:00 AM
                  </option>
                  <option
                    value="2"
                  >
                    02:00 AM
                  </option>
                  <option
                    value="3"
                  >
                    03:00 AM
                  </option>
                  <option
                    value="4"
                  >
                    04:00 AM
                  </option>
                  <option
                    value="5"
                  >
                    05:00 AM
                  </option>
                  <option
                    value="6"
                  >
                    06:00 AM
                  </option>
                  <option
                    value="7"
                  >
                    07:00 AM
                  </option>
                  <option
                    value="8"
                  >
                    08:00 AM
                  </option>
                  <option
                    value="9"
                  >
                    09:00 AM
                  </option>
                  <option
                    value="10"
                  >
                    10:00 AM
                  </option>
                  <option
                    value="11"
                  >
                    11:00 AM
                  </option>
                  <option
                    value="12"
                  >
                    12:00 AM
                  </option>
                  <option
                    value="13"
                  >
                    01:00 PM
                  </option>
                  <option
                    value="14"
                  >
                    02:00 PM
                  </option>
                  <option
                    value="15"
                  >
                    03:00 PM
                  </option>
                  <option
                    value="16"
                  >
                    04:00 PM
                  </option>
                  <option
                    value="17"
                  >
                    05:00 PM
                  </option>
                  <option
                    value="18"
                  >
                    06:00 PM
                  </option>
                  <option
                    value="19"
                  >
                    07:00 PM
                  </option>
                  <option
                    value="20"
                  >
                    08:00 PM
                  </option>
                  <option
                    value="21"
                  >
                    09:00 PM
                  </option>
                  <option
                    value="22"
                  >
                    10:00 PM
                  </option>
                  <option
                    value="23"
                  >
                    11:00 PM
                  </option>
                </select>
              </div>
            </div>
            <div
              class="lineContainerInformacion"
            >
              <div
                class="containerCityInput"
              >
                <label>
                  * Ciudad
                </label>
                <div
                  class="cityContainer divDrawer"
                >
                  
                  <span
                    class="gpsEmpty ocultar"
                  >
                    <i
                      class="fas fa-map-marker-alt"
                    />
                  </span>
                  <input
                    autocomplete="off"
                    class="input inputFormDatos"
                    placeholder="¿A dónde vamos?"
                    type="text"
                    value=""
                  />
                </div>
              </div>
              <div
                class="containerEstandarInput"
              >
                <label
                  class="labelEstandarInput"
                >
                  * País
                </label>
                <div
                  class="sombraConAutocompletado"
                >
                  <input
                    class="estandarInput"
                    name="pais"
                    required=""
                    type="text"
                    value=""
                  />
                </div>
              </div>
            </div>
            <div
              class="lineContainerInformacion"
            >
              <div
                class="containerNumberInput"
              >
                <label
                  class="labelNumberInput"
                >
                  Latitud
                </label>
                <input
                  class="numberInput"
                  name="latitud"
                  step="any"
                  type="number"
                  value=""
                />
              </div>
              <div
                class="containerNumberInput"
              >
                <label
                  class="labelNumberInput"
                >
                  Longitud
                </label>
                <input
                  class="numberInput"
                  name="longitud"
                  step="any"
                  type="number"
                  value=""
                />
              </div>
            </div>
            <div
              class="lineContainerInformacion"
            >
              <div
                class="containerTextArea"
              >
                <label>
                  Descripción
                </label>
                <textarea
                  cols="30"
                  id="descripcion"
                  name="descripcion"
                  rows="6"
                />
              </div>
            </div>
          </div>
          <div
            class="subContainer"
          >
            <h3
              class="subtituloBloque undefined"
            >
              Agregar atributos
            </h3>
            <ul
              class="checkboxes"
            >
              <li>
                <input
                  id="1"
                  name="caracteristicas"
                  type="checkbox"
                  value="Wifi"
                />
                <label
                  for="1"
                >
                  <i
                    class="fas fa-wifi"
                  />
                  <span>
                    Wifi
                  </span>
                </label>
              </li>
              <li>
                <input
                  id="2"
                  name="caracteristicas"
                  type="checkbox"
                  value="Piscina"
                />
                <label
                  for="2"
                >
                  <i
                    class="fas fa-swimmer"
                  />
                  <span>
                    Piscina
                  </span>
                </label>
              </li>
              <li>
                <input
                  id="3"
                  name="caracteristicas"
                  type="checkbox"
                  value="Estacionamiento gratuito"
                />
                <label
                  for="3"
                >
                  <i
                    class="fas fa-car"
                  />
                  <span>
                    Estacionamiento gratuito
                  </span>
                </label>
              </li>
              <li>
                <input
                  id="4"
                  name="caracteristicas"
                  type="checkbox"
                  value="Aire acondicionado"
                />
                <label
                  for="4"
                >
                  <i
                    class="fas fa-snowflake"
                  />
                  <span>
                    Aire acondicionado
                  </span>
                </label>
              </li>
              <li>
                <input
                  id="5"
                  name="caracteristicas"
                  type="checkbox"
                  value="No fumador"
                />
                <label
                  for="5"
                >
                  <i
                    class="fas fa-smoking-ban"
                  />
                  <span>
                    No fumador
                  </span>
                </label>
              </li>
              <li>
                <input
                  id="6"
                  name="caracteristicas"
                  type="checkbox"
                  value="Televisor"
                />
                <label
                  for="6"
                >
                  <i
                    class="fas fa-tv"
                  />
                  <span>
                    Televisor
                  </span>
                </label>
              </li>
              <li>
                <input
                  id="7"
                  name="caracteristicas"
                  type="checkbox"
                  value="Apto para mascotas"
                />
                <label
                  for="7"
                >
                  <i
                    class="fas fa-paw"
                  />
                  <span>
                    Apto para mascotas
                  </span>
                </label>
              </li>
              <li>
                <input
                  id="8"
                  name="caracteristicas"
                  type="checkbox"
                  value="Barra libre"
                />
                <label
                  for="8"
                >
                  <i
                    class="fas fa-cocktail"
                  />
                  <span>
                    Barra libre
                  </span>
                </label>
              </li>
              <li>
                <input
                  id="9"
                  name="caracteristicas"
                  type="checkbox"
                  value="Tarjeta de crédito"
                />
                <label
                  for="9"
                >
                  <i
                    class="fas fa-credit-card"
                  />
                  <span>
                    Tarjeta de crédito
                  </span>
                </label>
              </li>
              <li>
                <input
                  id="10"
                  name="caracteristicas"
                  type="checkbox"
                  value="Check-in"
                />
                <label
                  for="10"
                >
                  <i
                    class="fas fa-clock"
                  />
                  <span>
                    Check-in
                  </span>
                </label>
              </li>
            </ul>
          </div>
          <div
            class="subContainer"
          >
            <h3
              class="subtituloBloque undefined"
            >
              Políticas del producto
            </h3>
            <div
              class="bloquePoliticas"
            >
              <div
                class="columnPolitica"
              >
                <h3
                  class="tipoPolitica"
                >
                  Normas de la casa
                </h3>
                <div
                  class="containerTextArea"
                >
                  <label>
                    Descripción
                  </label>
                  <textarea
                    cols="30"
                    id="normasDeLaCasa"
                    name="normasDeLaCasa"
                    placeholder="Escribir aquí"
                    rows="6"
                  />
                </div>
              </div>
              <div
                class="columnPolitica"
              >
                <h3
                  class="tipoPolitica"
                >
                  Salud y seguridad
                </h3>
                <div
                  class="containerTextArea"
                >
                  <label>
                    Descripción
                  </label>
                  <textarea
                    cols="30"
                    id="saludSeguridad"
                    name="saludSeguridad"
                    placeholder="Escribir aquí"
                    rows="6"
                  />
                </div>
              </div>
              <div
                class="columnPolitica"
              >
                <h3
                  class="tipoPolitica"
                >
                  Política de cancelación
                </h3>
                <div
                  class="containerTextArea"
                >
                  <label>
                    Descripción
                  </label>
                  <textarea
                    cols="30"
                    id="cancelacion"
                    name="cancelacion"
                    placeholder="Escribir aquí"
                    rows="6"
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            class="subContainer"
          >
            <h3
              class="subtituloBloque undefined"
            >
              Cargar imágenes
            </h3>
            <div
              class="container"
            >
              <div
                class="fila"
              >
                <div
                  class="containerEstandarInput"
                >
                  <label
                    class="labelEstandarInput"
                  />
                  <div
                    class="sombraConAutocompletado"
                  >
                    <input
                      class="estandarInput"
                      name="url"
                      placeholder="Insertar https://"
                      required=""
                      type="text"
                      value=""
                    />
                  </div>
                </div>
                <div
                  class="containerDropInput"
                >
                  <label
                    class="labelDropInput"
                  />
                  <div
                    class="cityContainer divDrawer"
                  >
                    
                    <input
                      autocomplete="off"
                      class="input"
                      placeholder="Descripción"
                      type="text"
                      value=""
                    />
                  </div>
                </div>
                <div
                  class="containerButton"
                >
                  <div
                    class="containerBotones"
                  >
                    <button
                      class="add"
                      disabled=""
                    >
                      <i
                        class="fas fa-check"
                      />
                    </button>
                    <button
                      class="add"
                    >
                      <i
                        class="fas fa-plus"
                      />
                    </button>
                  </div>
                </div>
              </div>
              
            </div>
          </div>
          <div
            class="subContainer"
          >
            <input
              class="filledButton buttonSubmit"
              type="submit"
              value="Crear"
            />
          </div>
        </form>
      </section>
    </div>
  </body>,
  "container": <div>
    <div
      class="contenedorHeader"
    >
      <div
        class="headerIzquierda"
      >
        <div
          class="a"
        >
          <span>
            <i
              aria-hidden="true"
              class="fas fa-chevron-left"
            />
          </span>
          <span>
            ATRÁS
          </span>
        </div>
      </div>
      <div
        class="headerDerecha"
      >
        <h3
          class="subtituloBloque favTitle"
        >
          Administración
        </h3>
      </div>
    </div>
    <section
      class="containerPrincipal"
    >
      <h3
        class="subtituloBloque undefined"
      >
        Crear propiedad
      </h3>
      <p
        class="camposObligatorios"
      >
        (Los campos identificados con * son obligatorios)
      </p>
      <form
        class="formAdmin"
      >
        <div
          class="subContainer"
        >
          <div
            class="lineContainerInformacion"
          >
            <div
              class="containerEstandarInput"
            >
              <label
                class="labelEstandarInput"
              >
                * Nombre de la propiedad
              </label>
              <div
                class="sombraConAutocompletado"
              >
                <input
                  class="estandarInput"
                  name="nombre"
                  required=""
                  type="text"
                  value=""
                />
              </div>
            </div>
            <div
              class="containerSelect"
            >
              <label>
                * Categoría
              </label>
              <select
                class="minimal"
                name="tituloCategoria"
                required=""
              >
                <option
                  class="optionDefault"
                  hidden=""
                  value="tituloCategoria"
                />
              </select>
            </div>
          </div>
          <div
            class="lineContainerInformacion"
          >
            <div
              class="containerEstandarInput"
            >
              <label
                class="labelEstandarInput"
              >
                * Dirección
              </label>
              <div
                class="sombraConAutocompletado"
              >
                <input
                  class="estandarInput"
                  name="direccion"
                  required=""
                  type="text"
                  value=""
                />
              </div>
            </div>
            <div
              class="containerSelect"
            >
              <label>
                * CheckIn (horario min)
              </label>
              <select
                class="minimal"
                name="horarioCheckIn"
                required=""
              >
                <option
                  class="optionDefault"
                  hidden=""
                  value="horarioCheckIn"
                />
                <option
                  value="0"
                >
                  00:00 AM
                </option>
                <option
                  value="1"
                >
                  01:00 AM
                </option>
                <option
                  value="2"
                >
                  02:00 AM
                </option>
                <option
                  value="3"
                >
                  03:00 AM
                </option>
                <option
                  value="4"
                >
                  04:00 AM
                </option>
                <option
                  value="5"
                >
                  05:00 AM
                </option>
                <option
                  value="6"
                >
                  06:00 AM
                </option>
                <option
                  value="7"
                >
                  07:00 AM
                </option>
                <option
                  value="8"
                >
                  08:00 AM
                </option>
                <option
                  value="9"
                >
                  09:00 AM
                </option>
                <option
                  value="10"
                >
                  10:00 AM
                </option>
                <option
                  value="11"
                >
                  11:00 AM
                </option>
                <option
                  value="12"
                >
                  12:00 AM
                </option>
                <option
                  value="13"
                >
                  01:00 PM
                </option>
                <option
                  value="14"
                >
                  02:00 PM
                </option>
                <option
                  value="15"
                >
                  03:00 PM
                </option>
                <option
                  value="16"
                >
                  04:00 PM
                </option>
                <option
                  value="17"
                >
                  05:00 PM
                </option>
                <option
                  value="18"
                >
                  06:00 PM
                </option>
                <option
                  value="19"
                >
                  07:00 PM
                </option>
                <option
                  value="20"
                >
                  08:00 PM
                </option>
                <option
                  value="21"
                >
                  09:00 PM
                </option>
                <option
                  value="22"
                >
                  10:00 PM
                </option>
                <option
                  value="23"
                >
                  11:00 PM
                </option>
              </select>
            </div>
          </div>
          <div
            class="lineContainerInformacion"
          >
            <div
              class="containerCityInput"
            >
              <label>
                * Ciudad
              </label>
              <div
                class="cityContainer divDrawer"
              >
                
                <span
                  class="gpsEmpty ocultar"
                >
                  <i
                    class="fas fa-map-marker-alt"
                  />
                </span>
                <input
                  autocomplete="off"
                  class="input inputFormDatos"
                  placeholder="¿A dónde vamos?"
                  type="text"
                  value=""
                />
              </div>
            </div>
            <div
              class="containerEstandarInput"
            >
              <label
                class="labelEstandarInput"
              >
                * País
              </label>
              <div
                class="sombraConAutocompletado"
              >
                <input
                  class="estandarInput"
                  name="pais"
                  required=""
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
          <div
            class="lineContainerInformacion"
          >
            <div
              class="containerNumberInput"
            >
              <label
                class="labelNumberInput"
              >
                Latitud
              </label>
              <input
                class="numberInput"
                name="latitud"
                step="any"
                type="number"
                value=""
              />
            </div>
            <div
              class="containerNumberInput"
            >
              <label
                class="labelNumberInput"
              >
                Longitud
              </label>
              <input
                class="numberInput"
                name="longitud"
                step="any"
                type="number"
                value=""
              />
            </div>
          </div>
          <div
            class="lineContainerInformacion"
          >
            <div
              class="containerTextArea"
            >
              <label>
                Descripción
              </label>
              <textarea
                cols="30"
                id="descripcion"
                name="descripcion"
                rows="6"
              />
            </div>
          </div>
        </div>
        <div
          class="subContainer"
        >
          <h3
            class="subtituloBloque undefined"
          >
            Agregar atributos
          </h3>
          <ul
            class="checkboxes"
          >
            <li>
              <input
                id="1"
                name="caracteristicas"
                type="checkbox"
                value="Wifi"
              />
              <label
                for="1"
              >
                <i
                  class="fas fa-wifi"
                />
                <span>
                  Wifi
                </span>
              </label>
            </li>
            <li>
              <input
                id="2"
                name="caracteristicas"
                type="checkbox"
                value="Piscina"
              />
              <label
                for="2"
              >
                <i
                  class="fas fa-swimmer"
                />
                <span>
                  Piscina
                </span>
              </label>
            </li>
            <li>
              <input
                id="3"
                name="caracteristicas"
                type="checkbox"
                value="Estacionamiento gratuito"
              />
              <label
                for="3"
              >
                <i
                  class="fas fa-car"
                />
                <span>
                  Estacionamiento gratuito
                </span>
              </label>
            </li>
            <li>
              <input
                id="4"
                name="caracteristicas"
                type="checkbox"
                value="Aire acondicionado"
              />
              <label
                for="4"
              >
                <i
                  class="fas fa-snowflake"
                />
                <span>
                  Aire acondicionado
                </span>
              </label>
            </li>
            <li>
              <input
                id="5"
                name="caracteristicas"
                type="checkbox"
                value="No fumador"
              />
              <label
                for="5"
              >
                <i
                  class="fas fa-smoking-ban"
                />
                <span>
                  No fumador
                </span>
              </label>
            </li>
            <li>
              <input
                id="6"
                name="caracteristicas"
                type="checkbox"
                value="Televisor"
              />
              <label
                for="6"
              >
                <i
                  class="fas fa-tv"
                />
                <span>
                  Televisor
                </span>
              </label>
            </li>
            <li>
              <input
                id="7"
                name="caracteristicas"
                type="checkbox"
                value="Apto para mascotas"
              />
              <label
                for="7"
              >
                <i
                  class="fas fa-paw"
                />
                <span>
                  Apto para mascotas
                </span>
              </label>
            </li>
            <li>
              <input
                id="8"
                name="caracteristicas"
                type="checkbox"
                value="Barra libre"
              />
              <label
                for="8"
              >
                <i
                  class="fas fa-cocktail"
                />
                <span>
                  Barra libre
                </span>
              </label>
            </li>
            <li>
              <input
                id="9"
                name="caracteristicas"
                type="checkbox"
                value="Tarjeta de crédito"
              />
              <label
                for="9"
              >
                <i
                  class="fas fa-credit-card"
                />
                <span>
                  Tarjeta de crédito
                </span>
              </label>
            </li>
            <li>
              <input
                id="10"
                name="caracteristicas"
                type="checkbox"
                value="Check-in"
              />
              <label
                for="10"
              >
                <i
                  class="fas fa-clock"
                />
                <span>
                  Check-in
                </span>
              </label>
            </li>
          </ul>
        </div>
        <div
          class="subContainer"
        >
          <h3
            class="subtituloBloque undefined"
          >
            Políticas del producto
          </h3>
          <div
            class="bloquePoliticas"
          >
            <div
              class="columnPolitica"
            >
              <h3
                class="tipoPolitica"
              >
                Normas de la casa
              </h3>
              <div
                class="containerTextArea"
              >
                <label>
                  Descripción
                </label>
                <textarea
                  cols="30"
                  id="normasDeLaCasa"
                  name="normasDeLaCasa"
                  placeholder="Escribir aquí"
                  rows="6"
                />
              </div>
            </div>
            <div
              class="columnPolitica"
            >
              <h3
                class="tipoPolitica"
              >
                Salud y seguridad
              </h3>
              <div
                class="containerTextArea"
              >
                <label>
                  Descripción
                </label>
                <textarea
                  cols="30"
                  id="saludSeguridad"
                  name="saludSeguridad"
                  placeholder="Escribir aquí"
                  rows="6"
                />
              </div>
            </div>
            <div
              class="columnPolitica"
            >
              <h3
                class="tipoPolitica"
              >
                Política de cancelación
              </h3>
              <div
                class="containerTextArea"
              >
                <label>
                  Descripción
                </label>
                <textarea
                  cols="30"
                  id="cancelacion"
                  name="cancelacion"
                  placeholder="Escribir aquí"
                  rows="6"
                />
              </div>
            </div>
          </div>
        </div>
        <div
          class="subContainer"
        >
          <h3
            class="subtituloBloque undefined"
          >
            Cargar imágenes
          </h3>
          <div
            class="container"
          >
            <div
              class="fila"
            >
              <div
                class="containerEstandarInput"
              >
                <label
                  class="labelEstandarInput"
                />
                <div
                  class="sombraConAutocompletado"
                >
                  <input
                    class="estandarInput"
                    name="url"
                    placeholder="Insertar https://"
                    required=""
                    type="text"
                    value=""
                  />
                </div>
              </div>
              <div
                class="containerDropInput"
              >
                <label
                  class="labelDropInput"
                />
                <div
                  class="cityContainer divDrawer"
                >
                  
                  <input
                    autocomplete="off"
                    class="input"
                    placeholder="Descripción"
                    type="text"
                    value=""
                  />
                </div>
              </div>
              <div
                class="containerButton"
              >
                <div
                  class="containerBotones"
                >
                  <button
                    class="add"
                    disabled=""
                  >
                    <i
                      class="fas fa-check"
                    />
                  </button>
                  <button
                    class="add"
                  >
                    <i
                      class="fas fa-plus"
                    />
                  </button>
                </div>
              </div>
            </div>
            
          </div>
        </div>
        <div
          class="subContainer"
        >
          <input
            class="filledButton buttonSubmit"
            type="submit"
            value="Crear"
          />
        </div>
      </form>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
